Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPINamespaceCocoa.mm

```diff
@@ -74,6 +74,16 @@
     if (name == "pageAction"_s)
         return !extensionContext().supportsManifestVersion(3) && objectForKey<NSDictionary>(extensionContext().manifest(), @"page_action", false);
 
+#if ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)
+    // TODO: check if sidebar feature is enabled and if we have a sidebarAction entry in the manifest to determine if we can use sidebarAction
```

Comment:
nit: we usually use `// FIXME: rdar<123456>  Check if sidebar feature is enabled ...`

Reply:
or if this is being tracked bug.webkit.org, then the link to the webkit bug